bite-size-notes/Week3-c_TrajectoryOptimization_Devansh.md

```diff
@@ -0,0 +1,133 @@
+## Trajectory Optimization - Linear Systems
+# Objectives
+1. How CBF's and CLF's contribute towards Trajectory Optimization.
+2. Convex Formulation for Linear System:
+
+   a. Continuous Time
+  
+   b. Discrete Time
+
+## Introduction
+
+Traditional approaches of acheiving optimal control with a feedback controller has a lot of limitations, such as they do not work well for systems with state dimensions above 4 or 5, effectiveness of the controller is limited to the specific region of state space where linearization is an approximation of non-linear dynamics and so on. Trajectory Optimization is an essential computational tool to formulate a simpler version of optimization problem by linearizing over a nominal operation point(tajectory), attempting to find optimal control solution which is valid only form a single initial condition rather than finding a solution for the entire state space fro a feedback controller. For this to happen, we represent this solution as a trajectory, x(t), u(t) , defined over a finite interval 'tf'. 
+
```

Comment:
I like the clear and concise introduction, there are a couple of typo errors though.

Reply:
Thanks, Resolved the typos.